testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -100,7 +100,7 @@ protected abstract QueryRunner createQueryRunner()
             throws Exception;
 
     @AfterClass(alwaysRun = true)
-    public void close()
+    public final void close()
```

Comment:
Maybe we could more uncommon method name here. Maybe `closeTestQueryFramework()`?

Reply:
i always call clean-up methods  `cleanUp` if i don't give them a more specific name.
`close` *is* uncommon in this context.